safekeeper.pid

```diff
@@ -0,0 +1 @@
+68975
```

Comment:
Looks like it was committed by accident?

Reply:
true, just removed it.